torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -416,5 +421,85 @@ GraphExecutorState ProfilingGraphExecutorImpl::getDebugState() {
   return state;
 }
 
+Node* createFallbackGraph(Block* b, ArrayRef<Value*> inputs, Graph* g) {
+  auto graph = std::make_shared<Graph>();
```

Comment:
We might want to assert that `!b->owningNode() || b->owningNode()->kind() == prim::If` - it would invalid to run this on a loop block

Reply:
a good point!

Reply:
actually, dont think this invalid to run on a loop block it would just be weird lol. so no assertion is fine...

Reply:
yup it's perfectly valid; sorry i should've posted earlier when i revisited your suggestion